### PR TITLE
Update Flotiq entry

### PR DESCRIPTION
**Pull request by andrzejwp**
Please accept this PR, which provides an updated description for Flotiq

Thank you :)

### CODE REVIEW

**Comment by andrzejwp** [comment]
And an semi-automated shout-out to @philhawksworth  :-)

**Comment by andrzejwp** [comment]
Conflicts resolved.

File: content/projects/flotiq.md [hunk 1]
```diff
@@ -1,34 +1,102 @@
 ---
 title: Flotiq
+repo: flotiq/flotiq-blog
 homepage: https://flotiq.com/
+repo: flotiq/flotiq-blog
```
Comment (philhawksworth) [inline content/projects/flotiq.md:5]: Looks like this line is duplicated. That halts the site build.
If you de-dupe, I should be able to merge this

Comment (andrzejwp) [reply content/projects/flotiq.md:4]: Fixed, sorry about that.

**Comment by andrzejwp** [comment]
Removed the duplicated entry. Thank you.

**Pull request merged**